src/Resources/contao/library/Contao/Template.php

```diff
@@ -265,7 +265,7 @@ public function output()
 
 		header('Vary: User-Agent', false);
 		header('Content-Type: ' . $this->strContentType . '; charset=' . \Config::get('characterSet'));
-
+		 // FIXME: would be cool to throw a response exception here but sadly the hook is in the way.
```

Comment:
I remember that we agreed that it is but not marked as such in the code yet - changed in e408bf2.